src/current/v24.1/migration-overview.md

```diff
@@ -62,17 +62,17 @@ A lift-and-shift approach is the most straightforward. However, it's important t
 
 - *Reduced functionality* takes some, but not all, application functionality offline. For example, you can disable writes but not reads while you migrate the application data, and queue data to be written after completing the migration.
 
-For an overview of lift-and-shift migrations to CockroachDB, see [Lift and Shift](#lift-and-shift). For considerations and details about the pros and cons of this approach, see [Migration Strategy: Lift and Shift]({% link {{ page.version.version }}/migration-strategy-lift-and-shift.md %}).
+For an overview of lift-and-shift migrations to CockroachDB, see [Lift and Shift](#lift-and-shift).
 
 #### Minimal downtime
 
-If your application cannot tolerate downtime, then you should aim for a "zero-downtime" approach. "Zero" means that downtime is reduced to either an absolute minimum or zero, such that users do not notice the migration.
+If your application cannot tolerate downtime, then you should aim for a *live migration*. This is also called a "zero-downtime" approach. "Zero" means that downtime is reduced to either an absolute minimum or zero, such that users do not notice the migration.
```

Comment:
I'm still unsure if it's ok to call `zero-downtime`, since we don't advocate immediate cutover. And should we actually present immediate cutover here? cc @Jeremyyang920 @ajstorm

Reply:
This is a question also for PM - I can address in a separate PR, but I have simplified these sentences in the meantime.

Reply:
I think it might be okay to present it since it really is an option out there. But, im not sure we want to term is live migration to confuse with the LMS itself. Again part of the whole renaming and branding exercise we want to do. 

I think we should just call out that "zero" can exist but there are caveats such as data consistency and leave it at that.